tests/pytest_translate

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+#
+# Translate a python test to another language.
+#
+import enum
+import sys
+import os
+import argparse
+import ast
+
+sys.path.insert(0, os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), "translate"))
+
+# Not sure how to solve thie pep8 error:
+#   pytest_translate:17:1: E402 module level import not at top of file
+# this cannot be imported until we add the import path above
+import pythonir
+
+parser = argparse.ArgumentParser(
```

Comment:
It'd be nice if there were hardly any file-scope statements. I'm aware this is common for scripts, but the likelihood that this code will grow beyond a simple script is incredibly high. It'd be nice if the script part of the code was just calling a simple main function.